src/test/scala/htsjdk/samtools/util/StringUtilTest.scala

```diff
@@ -0,0 +1,111 @@
+package htsjdk.samtools.util
+
+import htsjdk.UnitSpec
+
+class StringUtilTest extends UnitSpec {
+  "StringUtil.split" should "should behave like String.split(char)" in {
```

Comment:
This seems to be a place where the reverse order makes more sence, since you are using the same data for two tests...wouldn't it make better sense to have 

data.foreach(s=> test1; test2) ?